Apps/WebClient/src/ClientApp/src/components/calendar/calendar.vue

```diff
@@ -119,6 +115,8 @@ export default class CalendarComponent extends Vue {
                     ) {
                         groups.push(monthYear);
                     }
+                } else {
+                    console.log("Invalid entry date:", entry);
```

Comment:
In case of invalid date, it is probably better to log as a warning instead of info here.

Reply:
Right now we don't have the capabilities to differentiate the log level. We only have console.log.

Reply:
how about using "console.warn" instead of "console.log" as it is a warning not info.

Reply:
we could, but we are going to go over all logs anyhow in an upcoming story

Reply:
sounds good to me, I would like to remove this and add only as a part of the upcoming stories then, but it shouldn't be a blocker.

Reply:
I don't think removing it is a good idea. If the date is invalid we would have other sets of issues.

Reply:
so why don't we use console.warn though as you want to put this code in anyway? I think it is better to have different levels of logs right. These details will be useful for us to refactor code later on.